### PR TITLE
Fix AWS SQS API version and casing

### PR DESCRIPTION
**Pull request by ivov**
This PR adds the API version to the AWS SQS node to prevent selective enforcement of the version requirement.

Casing needs to be adjusted when locking the API version.

### CODE REVIEW

File: packages/nodes-base/nodes/Aws/SQS/AwsSqs.node.ts [hunk 1]
```diff
@@ -373,7 +373,8 @@ export class AwsSqs implements INodeType {
 
 			let responseData;
 			try {
-				responseData = await awsApiRequestSOAP.call(this, 'sqs', 'GET', `${queuePath}/?Action=${operation}&` + params.join('&'));
+				const properCasedOperation = operation[0].toUpperCase() + operation.slice(1);
+				responseData = await awsApiRequestSOAP.call(this, 'sqs', 'GET', `${queuePath}/?Version=2012-11-05&Action=${properCasedOperation}&` + params.join('&'));
```
Comment (mutdmour) [inline packages/nodes-base/nodes/Aws/SQS/AwsSqs.node.ts:377]: we should apply this version to loading queues as well https://github.com/n8n-io/n8n/blob/48690d17c13edf49e0f32e28e8a51673e00118ed/packages/nodes-base/nodes/Aws/SQS/AwsSqs.node.ts#L272

Comment (mutdmour) [inline packages/nodes-base/nodes/Aws/SQS/AwsSqs.node.ts:377]: nitpick - we should move these two params to params array [here](https://github.com/n8n-io/n8n/blob/48690d17c13edf49e0f32e28e8a51673e00118ed/packages/nodes-base/nodes/Aws/SQS/AwsSqs.node.ts#L311 ) to follow existing patterns like [here](https://github.com/n8n-io/n8n/blob/master/packages/nodes-base/nodes/Aws/AwsSnsTrigger.node.ts#L110)

**Review by mutdmour — COMMENTED** [review]
thank you for making this fix 🙏

**Comment by ivov** [comment]
@mutdmour Thanks for your feedback! Both points addressed.

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with n8n@0.116.0

**Pull request merged**